.github/workflows/audio-integration.yml

```diff
@@ -19,37 +19,90 @@
       runs-on: ubuntu-latest
 
       steps:
+        - name: Checkout Package
+          uses: actions/checkout@v2
+          with:
+            fetch-depth: 0
+        - name: Check if needed to run
+          id: test_needed
+          run: |
+            base_branch=$(head -n 1 ${GITHUB_WORKSPACE}/.base-branch)
```

Comment:
Looks like this is getting repeated in each test, can we somehow have a common script and run that across tests?

Reply:
Updated.